runtime/tests/port/si_numcpusTest.c

```diff
@@ -117,69 +89,6 @@ int j9sysinfo_numcpus_test4 (J9PortLibrary* portLibrary, UDATA boundTest)
 	return reportTestExit(portLibrary, testName);
 }
 
-/**
- * Set entitled = number of physical CPUs. get target number of CPUs.
- * Validate target is OMR_MIN(physical, bound).
- *
- * @param[in] portLibrary The port library under test
- *
- * @return TEST_PASSED on success, TEST_FAILED on failure
- */
-int j9sysinfo_numcpus_test5 (J9PortLibrary* portLibrary)
-{
-	PORT_ACCESS_FROM_PORT(portLibrary);
-	const char* testName = "j9sysinfo_numcpus_test5";
-	UDATA numberPhysicalCPUs = j9sysinfo_get_number_CPUs_by_type(J9PORT_CPU_PHYSICAL);
-	UDATA numberBoundCPUs = j9sysinfo_get_number_CPUs_by_type(J9PORT_CPU_BOUND);
-	UDATA numberEntitledCPUs = j9sysinfo_get_number_CPUs_by_type(J9PORT_CPU_ENTITLED);
-	UDATA targetNumberCPUs = j9sysinfo_get_number_CPUs_by_type(J9PORT_CPU_TARGET);
-	UDATA minimum = 0;
-
-	j9sysinfo_set_number_entitled_CPUs(numberPhysicalCPUs);
-
-	outputComment(PORTLIB,"\nSet number of entitled CPUs to number of physical CPUs.\n");
-	outputComment(PORTLIB,"Get target number of CPUs.\n");
-	outputComment(PORTLIB,"	Expected: OMR_MIN(%d, %d)\n", numberEntitledCPUs, numberBoundCPUs);
-	outputComment(PORTLIB,"	Result:   %d\n", targetNumberCPUs);
-
-	/* Get minimum of numberEntitledCPUs and numberBoundCPUs */
-	minimum = OMR_MIN(numberEntitledCPUs, numberBoundCPUs);
-
-	if (minimum != targetNumberCPUs) {
```

Comment:
I don't believe this test should be removed. It can still check targetNumberCPUs == numberBoundCPUs.